packages/jest-runtime/src/transform.js

```diff
@@ -293,6 +294,30 @@ const transformSource = (
   return result;
 };
 
+const generateEmptyCoverage = (
+  filename: Path,
+  config: Config,
+  content: string
+): Object => {
+  const transform = getTransformer(filename, config);
+  let source = content;
+
+  if (shouldTransform(filename, config)) {
+    if (transform.canInstrument && typeof transform.getEmptyCoverage === 'function') {
+      return transform.getEmptyCoverage(content, filename, config, {
```

Comment:
does that delegate getting the empty coverage object to a transformer if it implements it?